src/screen/home.js

```diff
@@ -85,8 +95,31 @@ function App() {
         "&appid=" +
         process.env.REACT_APP_APIKEY
     );
+    const bookmarkedLocations = JSON.parse(localStorage.getItem('bookmarkedLocations'));
+    if(bookmarkedLocations.includes(city)) setBookmarked(true);
+    else setBookmarked(false);
   }, [city]);
 
+  useEffect(()=>{
```

Comment:
delete this entirely if this does not serve any purpose in prod

Reply:
removed